design/mapped_attribute.go

```diff
@@ -152,3 +152,16 @@ func (ma *MappedAttributeExpr) Merge(other *MappedAttributeExpr) {
 		}
 	}
 }
+
+// FindKey finds the given key in the mapped attribute expression.
```

Comment:
Is this supposed to be in this PR?

Reply:
I can create a separate PR if you would like. Just thought I would add this here so that we can fix the plugins build.